gcloud/bigtable/client.py

```diff
@@ -375,3 +378,30 @@ def cluster(self, zone, cluster_id, display_name=None, serve_nodes=3):
         """
         return Cluster(zone, cluster_id, self,
                        display_name=display_name, serve_nodes=serve_nodes)
+
+    def list_zones(self, timeout_seconds=None):
+        """Lists zones associated with project.
+
+        :type timeout_seconds: int
+        :param timeout_seconds: Number of seconds for request time-out.
+                                If not passed, defaults to value set on client.
+
+        :rtype: list
+        :returns: The names (as :class:`str`) of the zones
+        :raises: :class:`ValueError <exceptions.ValueError>` if one of the
+                 zones is not in ``OK`` state.
+        """
+        request_pb = messages_pb2.ListZonesRequest(name=self.project_name)
+        timeout_seconds = timeout_seconds or self.timeout_seconds
```

Comment:
:+1: Good call.

If you recall in our design review (https://github.com/dhermes/gcloud-python-bigtable/issues/9) you suggested passing a `Client` there instead of just the `timeout_seconds`.

Honestly, I'd rather not even have an override and the only reason I gave one was to match the style of the rest of `gcloud-python`.

WDYT about just ditching the override altogether?